content/kotsadm/installing/airgap-packages.md

```diff
@@ -8,6 +8,34 @@ draft: false
 
 When installing an application from an airgap package, the container images and application manifests are provided by the application vendor in an archive that can be used to deliver the artifacts into the cluster. This feature is only available for licenses that have the airgap feature enabled.
 
+## Kots install
+Begin by [installing the Kots CLI kubectl plugin](/kots-cli/getting-started/).  The Admin Console can be installed using the CLI and the airgap package included in the [release assets](https://github.com/replicatedhq/kots/releases) named `kotsadm.tar.gz`.   The asset version has to match the CLI version, which can be determined by running:
```

Comment:
Begin by [installing the KOTS kubectl plugin](/kots-cli/getting-started/). 
The Admin Console can be installed using the KOTS plugin and the airgap package that can be downloaded from the [release assets](https://github.com/replicatedhq/kots/releases/latest) named `kotsadm.tar.gz`. 
The asset version must match the KOTS CLI version, which can be determined by running: